config/locales/zh-CN.yml

```diff
@@ -524,10 +524,6 @@ zh-CN:
     pinned: 置顶嘟文
     reblogged: 转嘟
     sensitive_content: 敏感内容
-  support:
-    array:
-      last_word_connector: 以及
-      words_connector: "、"
```

Comment:
Since the authorization page is the only place that used `to_sentence`, can we include the `support.array` key, which overrides the existing one from rails-i18n, for better wording? Not sure about patching this to rails-i18n directly since `以及` might sound weird in some occasions.

Reply:
We should probably keep this override. Are they using the backwards comma (、) correctly in rails-i18n? If not let's keep overriding them and maybe send a bug report.

Reply:
@Artoria2e5 Nope, [here's the file](https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/zh-CN.yml). We need a developer's help to keep this override.

Reply:
Pull request submitted at svenfuchs/rails-i18n#756

Reply:
Fixed at upstream